pkgs/development/python-modules/httplib2/default.nix

```diff
@@ -1,21 +1,53 @@
-{ lib, buildPythonPackage, fetchPypi }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, mock
+, pyparsing
+, pytest-forked
+, pytest-randomly
+, pytest-timeout
+, pytest-xdist
+, pytestCheckHook
+, six
+}:
 
 buildPythonPackage rec {
   pname = "httplib2";
-  version = "0.18.1";
+  version = "0.19.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "8af66c1c52c7ffe1aa5dc4bcd7c769885254b0756e6e69f953c7f0ab49a70ba3";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04y2bc2yv3q84llxnafqrciqxjqpxbrd8glbnvvr16c20fwc3r4q";
   };
 
-  # Needs setting up
-  doCheck = false;
+  postPatch = ''
+    sed -i "/--cov/d" setup.cfg
```

Comment:
Should we generally remove pytset-cov or is this a special case here?

Reply:
I'm following @mweinelt's lead here. He suggested to remove it. I agree that the coverage is something that has value for the project itself but not for us.

I think that we should definitely discuss the removal of `--cov` and its friends by default.

Reply:
> Should we generally remove pytset-cov or is this a special case here?

If you have a general idea on how we can remove this generally, let's hear it.

Reply:
I think patching this would be a bigger project because you can enable pytest-cov in quite a few places and ways. Maybe an easier approach would be to write a dummy pytest plugin which accepts all pytest-cov options and does a no-op with it. If it does not hurt execution time we can also include it in the packages to keep it simple.

Reply:
> Example we could patch it from setup.cfg like https://github.com/NixOS/nixpkgs/pull/113014/files#diff-5203d6935690aad5c62a27e9f2371bb3a7032a0cb1425adaf8439890b266ae75R25-R27 but there are other way to enable it.

That only works if the `--cov` option is on a dedicated line, which very often is not the case.